react/src/helper/index.tsx

```diff
@@ -396,3 +396,32 @@ export function formatToUUID(str: string) {
 export const toGlobalId = (type: string, id: string): string => {
   return btoa(`${type}:${id}`);
 };
+
+/**
+ * Custom startCase function that preserves specified separators.
+ * @param {string} string - The input string to transform.
+ * @param {string|RegExp} [separator='.'] - The separator(s) to preserve.
+ * @returns {string} The transformed string.
+ */
+export const maintainSeparatorsAndStartCase = (
```

Comment:
I just added it and used the `preserveDotStartCase`  instead of `maintainSeparatorsAndStartCase`. Should I delete `maintainSeparatorsAndStartCase` also?

Reply:
I think it's better.